recipe/install_rdkit_dev.bat

```diff
@@ -0,0 +1,11 @@
+echo "Copying libs and headers"
+
+if not exist "%LIBRARY_LIB%" mkdir %LIBRARY_LIB%
+if not exist "%LIBRARY_INC%" mkdir %LIBRARY_INC%
+
+REM copy .lib and .pdb files to LIBRARY_LIB
```

Comment:
comment says `.pdb` files, but no `.pdb` files are copied

Reply:
My mistake, the install script comes from [another package](https://github.com/nsimakov/wxpython-feedstock/blob/copying_headers_and_libs/recipe/install_wxwidgets.bat) and I forgot to update the comment